db/wide/db_wide_basic_test.cc

```diff
@@ -690,6 +691,603 @@ TEST_F(DBWideBasicTest, MergeEntity) {
   verify_merge_ops_post_compaction();
 }
 
+TEST_F(DBWideBasicTest, MergeV3WideColumnOutput) {
```

Comment:
Not a big deal, but I think you can parameterize the test, `TEST_P`, make the output type either WideColumn Or Plain and remove some duplicate lines of code.

Reply:
Parameterized tests might not be the best fit here since some of the code (e.g. the merge operator implementation) is very different in the two cases but we can definitely factor out some of the repeated code into helper methods.